build/localstack-init/01_create_poller_sns.sh

```diff
@@ -0,0 +1,2 @@
+echo "########### Creating SqsPollerTopic SNS ###########"
+awslocal sns create-topic --name SqsPollerTopic
```

Comment:
Why do we need these bash scripts?

If I got it right, we create queue for the sample solution using AWS SDK

Reply:
Yes, it's better to create it before because if we run multiple times, it fails (on creating queue or topic)